third_party/xla_client/xrt_computation_client.cc

```diff
@@ -156,6 +158,17 @@ class TensorAllocator : public tensorflow::Allocator {
   std::unordered_map<AllocKey, AllocList::iterator, AllocKey::Hash> allocs_;
 };
 
+bool ShouldStartLocalService(const std::set<std::string>& devices) {
```

Comment:
We're also starting localservice for TPU VM right? I guess we're returning True here for TPUVM since we have `CPU:0` in TPU case too?

Reply:
Yea, that was my intention. For CPU and GPU we actually started multiple local server, but we only want to start one for the TPU setup.